### PR TITLE
Select video on pullup/edit

### PR DESCRIPTION
**Pull request by eldiegod**
fix: #590

### CODE REVIEW

**Review by drillprop — APPROVED** [review]
Looks good to me!

File: src/views/studio/MyVideosView/MyVideosView.tsx [hunk 1]
```diff
@@ -94,7 +94,8 @@ export const MyVideosView = () => {
       return
     }
     addVideoTab({ id, isDraft: opts.draft })
-
+    const tabIdx = videoTabs.findIndex((t) => t.id === id)
+    if (tabIdx >= 0) setSelectedVideoTabIdx(tabIdx)
```
Comment (kdembler) [inline src/views/studio/MyVideosView/MyVideosView.tsx]: We probably should call `addVideoTab` only if the tab is not found right?

**Review by kdembler — CHANGES_REQUESTED** [review]
I think this doesn't do exactly what we'd want to. In my head, "open in new tab" shouldn't select the tab, just open it. Only the edit should select it as well

File: src/views/studio/MyVideosView/MyVideosView.tsx (hunk 1 above)
Comment (eldiegod) [reply src/views/studio/MyVideosView/MyVideosView.tsx]: Not necessarily as the `addVideoTab` fn already checks for that so would be redundant

**Comment by eldiegod** [comment]
> want to. In my head, "open in new tab" shouldn't select the tab, just open it. Only the edit should select it as well

I think it wouldn't be an unexpected behavior in this case since the bar is minimized anyway so it doesn't switch the user's context. Although if we decide for the behaviour to be only open and not select then we run into the issue of having tabs open but nothing selected which crashes the app, wdyt? @kdembler

**Review by kdembler — APPROVED** [review]
Ok, let's not overcomplicate this

**Pull request merged**